e2e/test/scenarios/dashboard-filters/temporal-unit-parameters.cy.spec.js

```diff
@@ -779,6 +848,34 @@ describe("scenarios > dashboard > temporal unit parameters", () => {
       });
       getDashboardCard().findByText("Created At: Year").should("be.visible");
     });
+
+    it("should ignore invalid temporal unit values from the url", () => {
+      createDashboardWithMappedQuestion().then(dashboard => {
+        visitDashboard(dashboard.id, { params: { unit_of_time: "invalid" } });
+      });
+      filterWidget().within(() => {
+        cy.findByText(parameterDetails.name).should("be.visible");
+        cy.findByText(/invalid/i).should("not.exist");
+      });
+      getDashboardCard().findByText("Created At: Month").should("be.visible");
+    });
+
+    it("should accept temporal units outside of the allowlist if they are otherwise valid values from the url", () => {
```

Comment:
My understanding is that values which aren't whitelisted should not be allowed

Reply:
This is not the case - we discussed it with Product